src/pug/includes/head.pug

```diff
@@ -7,6 +7,14 @@ meta(http-equiv="X-UA-Compatible" content="IE=edge")
 
 meta(name="description" content="swag opportunities for developers")
 
+meta(property="og:title" content="devSwag")
+meta(property="og:description" content="Swag opportunities for developers")
```

Comment:
```suggestion
meta(property="og:description" content="swag opportunities for developers")
```

We either need to update the meta description, or keep this consistent 😄 ping @aslafy-z

Reply:
I prefer the lowercase one, @swapagarwal? 💯

Reply:
It's nice to have the `description` consistent. I have committed an update to use a lowercase `s` @vikaspotluri123 and @aslafy-z  for the suggestion.